css/styles.css

```diff
@@ -711,44 +717,23 @@ ul {
     box-sizing: border-box;
   }
 
-  #project1 .pimage {
-    width: 48.05%;
-    height: auto;
-    background-image: url("../images/p1imagebig.jpg");
-    background-size: contain;
-    background-repeat: no-repeat;
-    background-position: center;
-    transition: all 0.5s ease-out;
-  }
-
-  #project2 .pimage {
-    width: 48.05%;
-    height: auto;
-    background-image: url("../images/p1imagebig.jpg");
-    background-size: contain;
-    background-repeat: no-repeat;
-    background-position: center;
-    transition: all 0.5s ease-out;
-  }
-
-  #project3 .pimage {
+  .pimage {
     width: 48.05%;
     height: auto;
-    background-image: url("../images/p1imagebig.jpg");
     background-size: contain;
     background-repeat: no-repeat;
     background-position: center;
     transition: all 0.5s ease-out;
   }
 
-  #project4 .pimage {
-    width: 48.05%;
-    height: auto;
-    background-image: url("../images/p1imagebig.jpg");
-    background-size: contain;
+  .cimage {
+    width: 55%;
+    height: 47vh;
+    margin: 5px auto 15px auto;
+    align-self: center;
+    background-size: cover;
     background-repeat: no-repeat;
     background-position: center;
-    transition: all 0.5s ease-out;
   }
```

Comment:
Hello team ✋

- [ ] You may kindly fix the image on the desktop popup, as it appears stretched compared to the one provided in the Figma design.
- [ ]  You may also position the image properly as it seem to have wider margins on the left and right.
- Your design
![Your image](https://user-images.githubusercontent.com/63917725/180001272-6d61b9ce-c42a-41c9-bc35-47528c532197.PNG)

- Figma design
![Figma image](https://user-images.githubusercontent.com/63917725/180001353-70608c68-b306-4e9b-a2ee-23ae5c69c51d.PNG)